test/format/verify.c

```diff
@@ -234,6 +235,20 @@ table_verify_mirror(
       table->id, checkpoint == NULL ? "" : checkpoint, checkpoint == NULL ? "" : " checkpoint ");
     trace_msg(session, "%s: start", buf);
 
+    /*
+     * If we are not reading from a checkpoint, start a cursor to pin a page to ensure we never
+     * refresh our snapshot in verification.
+     */
+    if (checkpoint == NULL) {
+        wt_wrap_open_cursor(session, base->uri, NULL, &pinned_cursor);
+        pinned_ret = pinned_cursor->next(pinned_cursor);
```

Comment:
I might suggest adding a trace_msg here that we opened a pinned cursor for debugging.

Reply:
added